ts/session/apis/snode_api/factories/StoreGroupRequestFactory.ts

```diff
@@ -175,6 +184,8 @@ function makeStoreGroupConfigSubRequest({
             encryptedData: m.ciphertext,
             groupPk,
             secretKey: group.secretKey,
+            ttlMs: TTL_DEFAULT.CONFIG_MESSAGE,
+            getNow: NetworkTime.now,
```

Comment:
Why does this have to pass through to the class instead of called directly?

Reply:
this is mostly so we can share more stuff with swarmy.
If it isn't given in the constructor, an import is made in the requests file to NetworkTime.now(). On swarmy we don't need this, we can just use Date.now().
Passing it as  an arg means we can target session or swarmy with the same subrequest class

Reply:
Ah okay so we are decoupling the time so it's easier to use in both places. Got it 👍